js/jsonld.js

```diff
@@ -1646,6 +1646,30 @@ jsonld.cache = {
 };
 
 /**
+ * Accept header.
+ */
+jsonld.acceptHeader = 'application/ld+json, application/json';
```

Comment:
We should really have a `jsonld.defaults` object to put things like this under, if we don't have that already. If we don't have it, let's add it now before adding yet another new top-level thing:

```js
jsonld.defaults = {
  headers: {
    accept: 'application/ld+json, application/json'
  }
};
```

I wonder if it needs further scoping than that even. Thoughts from anyone else?